models/cohortdata.go

```diff
@@ -30,6 +32,8 @@ type PersonIdAndCohort struct {
 	CohortId int64
 }
 
+// This function returns the subjects that belong to both cohorts (the intersection of both cohorts)
+// TODO - name this function as such
```

Comment:
saw this TODO, is this just changing the function name? And if so should we just do it now?